### PR TITLE
[DOCU-3068] Portal Client API 

### PR DESCRIPTION
**Pull request by acgoldsmith**
Adding Portal Client API to API reference documentation list

### Description

DOCU-3068

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by rosskukulinski** [comment]
The official URL is now https://developer.konghq.com/spec/2aad2bcb-8d82-43b3-abdd-1d5e6e84dbd6/b4539157-4ced-4df5-affa-7d790baee356

**Comment by cloudjumpercat** [comment]
@rosskukulinski Thanks! I just added the URL in to the docs.

**Review by cloudjumpercat — APPROVED** [review]
LGTM! @rosskukulinski I'll let you merge this in once it's GA. Thanks!

File: app/konnect/api/index.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ The recommended method of authentication for {{site.konnect_short_name}} is [Per
 
 * [Runtime Configuration API](https://developer.konghq.com/spec/3c38bff8-3b7b-4323-8e2e-690d35ef97e0/) - Interface for creating and managing control plane certificates, data plane certificates, and {{site.konnect_short_name}} entities.
 
+* [Portal Client API](https://developer.konghq.com/spec/2aad2bcb-8d82-43b3-abdd-1d5e6e84dbd6/b4539157-4ced-4df5-affa-7d790baee356) - Interface for managing Dev Portal workflows via public APIs.
```
Comment (rosskukulinski) [inline app/konnect/api/index.md:35]: `Interface for building and integrating custom dev portal experiences powered by Konnect API data.

**Comment by rosskukulinski** [comment]
@acgoldsmith could you or someone from your team help with the merge conflicts?

**Pull request merged**